front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/StorageServiceSupport.java

```diff
@@ -244,6 +244,17 @@ public void bulkImport(Collection<T> items) {
     User authenticatedUser = new User();
     authenticatedUser.setUsername(AuthenticatedRequest.getSpinnakerUser().orElse("anonymous"));
 
+    if (service instanceof BulkStorageService) {
+      items.forEach(
+          item -> {
+            item.setLastModifiedBy(AuthenticatedRequest.getSpinnakerUser().orElse("anonymous"));
+            item.setLastModified(System.currentTimeMillis());
+          });
```

Comment:
prob should take out the values being set to outside the loop, faster and then update time will be same for all items
```suggestion
      long updateTime = System.currentTimeMillis();
      String modifiedBy = AuthenticatedRequest.getSpinnakerUser().orElse("anonymous");
      
      items.forEach(
          item -> {
            item.setLastModifiedBy(modifiedBy);
            item.setLastModified(updateTime);
          });
```

Reply:
Gotcha, good call ... I'll make this change.